src/OSPSuite.UI/Views/Charts/CurveColorGroupingView.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Assets;
+using OSPSuite.Presentation.Extensions;
+using OSPSuite.Presentation.Presenters.Charts;
+using OSPSuite.Presentation.Views.Charts;
+using OSPSuite.UI.Controls;
+
+namespace OSPSuite.UI.Views.Charts
+{
+   public partial class CurveColorGroupingView : BaseUserControl, ICurveColorGroupingView
+   {
+      private ICurveColorGroupingPresenter _presenter;
+
+      public CurveColorGroupingView()
+      {
+         InitializeComponent();
+         colorGroupingDescriptionLabelControl.Text = Captions.Chart.ColorGrouping.ColorGroupingDialogDescription.FormatForLabel();
+         applyColorGroupingButton.Text = Captions.Chart.ColorGrouping.ApplyColorGroupingButton;
+         applyColorGroupingButton.Click += (s, a) => OnEvent(onApplyColorGroupingButtonClicked, s, a);
```

Comment:
why are you passing s and a if they are not used in your method
And please call them (o, e) (object, event).

Reply:
done